app/view_models/GovernmentSpend.scala

```diff
@@ -34,11 +34,16 @@ case class GovernmentSpend(
   def sortedSpendData: List[(String, SpendData)] =
     govSpendAmountData.filter(_._1 != "GovSpendTotal").sortWith(_._2.percentage > _._2.percentage)
 
-  def filteredDataWithHigherTransport: List[(String, SpendData)] =
-    if (taxYear == 2019) {
+  def filteredDataWithHigherTransport: List[(String, SpendData)] = {
+    val listAfterFirstSwap = if (taxYear == 2019 || taxYear == 2020) {
```

Comment:
Do you want this to do a double swap? I think now it's getting a bit more complex, it would be better to rebuild the list in the correct order than rather do multiple swaps.